### DISCUSSION TITLE
Returns for all customers

### DISCUSSION THREAD

**Discussion by korberg** [PrestaShop/PrestaShop#32747] [Ideas]
### The problem

I was surprised that PrestaShop has "members only" return policy and with guest account the person is greeted with notice:

> You cannot return merchandise with a guest account.

At the same time there is no link provided to enroll in customer status at the same page.
Not only is this frustrating, but it is also against the law.
In EU there is 14 day withdrawal from transaction if it is made from online store. Link for further reading: [here](https://europa.eu/youreurope/citizens/consumers/shopping/guarantees-returns/index_en.htm#:~:text=If%20you%20bought%20a%20product,reason%20and%20without%20a%20justification.)

So in order to be compliant with EU regulations (and I do want to be compliant) I would need to add either such option manually or at least delete this notification. 

### The solution

By adding return options to ALL orders PrestaShop would be compliant with EU norms. 
If there are countries where such 'guest account' trick will work, let it then be selectable from Customer Service Return tab in Admin.

### Alternatives

The alternative method would be to create a dedicated page for returns and perhaps bypass built-in return function. Which would be a shame to do. After all - everything is already there, but it is just not by the regulations.

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
@PrestaShop/product-team

**Comment by Zoefinders** [comment]
Hello @korberg & @kpodemski 
@korberg thank for your proposal, we will check that with @PrestaShop/product-team and i come back to you ASAP
